youtube_dl/extractor/brightcove.py

```diff
@@ -489,34 +491,56 @@ def _extract_url(webpage):
     @staticmethod
     def _extract_urls(webpage):
         # Reference:
-        # 1. http://docs.brightcove.com/en/video-cloud/brightcove-player/guides/publish-video.html#setvideoiniframe
+        # 1. http://docs.brightcove.com/en/video-cloud/brightcove-player/guides/publish-video.html#tag
         # 2. http://docs.brightcove.com/en/video-cloud/brightcove-player/guides/publish-video.html#setvideousingjavascript
-        # 3. http://docs.brightcove.com/en/video-cloud/brightcove-player/guides/embed-in-page.html
-        # 4. https://support.brightcove.com/en/video-cloud/docs/dynamically-assigning-videos-player
+        # 3. http://docs.brightcove.com/en/video-cloud/brightcove-player/guides/publish-video.html#setvideoiniframe
```

Comment:
As explained in 15859c6845b158d80e6e3db94198d57602ef333c:
>  [Brightcove:new] reorder doc references in order of HTML page
> Otherwise it is very confusing to read them out of order.

That is, the links mostly to #portions of the same webpage, but it's confusing to try to read them in a different order than the webpage is in.